1.data-exploration/scripts/nbconverted/2.test-train_split.py

```diff
@@ -11,41 +11,49 @@
 sys.path.insert(0, "../0.data-download/scripts/")
 from data_loader import load_data
 from sklearn.model_selection import train_test_split
+import random
 
 
 # In[2]:
 
 
+random.seed(18)
+print(random.random())
+
+
+# In[3]:
+
+
 # load all of the data 
 data_directory = "../0.data-download/data/"
 dfs = load_data(data_directory, adult_or_pediatric = "all")
 dependency_df = dfs[1]
-sample_df = dfs[0]
+model_df = dfs[0]
```

Comment:
this is the pythonic way of doing this:

```python
model_df, dependency_df = load_data(data_directory, adult_or_pediatric = "all")
```

(it reduces three lines to one and reduces memory requirements by not duplicating data)